packages/controller/src/resources/loki/index.ts

```diff
@@ -196,22 +225,56 @@ export function LokiResources(
       ingestion_rate_mb: 500,
       ingestion_burst_size_mb: 2000,
       // enforced in querier
-      max_entries_limit_per_query: 60000 // Per-tenant entry limit per query
+      max_entries_limit_per_query: 60000, // Per-tenant entry limit per query
+      // https://github.com/grafana/loki/blob/main/production/ksonnet/loki/config.libsonnet#L190-L198
+      reject_old_samples: true,
```

Comment:
Forgot to link this article here
https://grafana.com/docs/loki/latest/design-documents/2021-01-ordering-constraint-removal/#flushing--chunk-creation